### PR TITLE
[Templates] Fix links to http://useiconic.com

### PR DESCRIPTION
**Pull request by javiercn**
Updates links to point to up-to-date documentation

## Description

Updates the links in the readme.md of one of our template files to point out to the newer documentation site.

## Customer Impact

Customers who will read this doc will have valid links to use for getting to the icons.

## Regression?

- [ ] Yes
- [x] No

[If yes, specify the version the behavior has regressed from]

## Risk

- [ ] High
- [ ] Medium
- [x] Low

It is updating a file in the templates that doesn't affect how the project is built or runs, only documentation.

## Verification

- [x] Manual (required)
- [ ] Automated

## Packaging changes reviewed?

- [ ] Yes
- [ ] No
- [x] N/A

### CODE REVIEW

File: src/Components/Samples/BlazorServerApp/wwwroot/css/open-iconic/README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-[Open Iconic v1.1.1](http://useiconic.com/open)
+[Open Iconic v1.1.1](https://ionic.io/ionicons)
 ===========
 
-### Open Iconic is the open source sibling of [Iconic](http://useiconic.com). It is a hyper-legible collection of 223 icons with a tiny footprint&mdash;ready to use with Bootstrap and Foundation. [View the collection](http://useiconic.com/open#icons)
+### Open Iconic is the open source sibling of [Iconic](http://useiconic.com). It is a hyper-legible collection of 223 icons with a tiny footprint&mdash;ready to use with Bootstrap and Foundation. [View the collection](https://ionic.io/ionicons)
```
Comment (mkArtakMSFT) [inline src/Components/Samples/BlazorServerApp/wwwroot/css/open-iconic/README.md:4]:
```suggestion
### Open Iconic is the open source sibling of [Iconic](https://iconic.io). It is a hyper-legible collection of 223 icons with a tiny footprint&mdash;ready to use with Bootstrap and Foundation. [View the collection](https://ionic.io/ionicons)
```

Comment (mkArtakMSFT) [reply src/Components/Samples/BlazorServerApp/wwwroot/css/open-iconic/README.md:4]: Also, is the name `ionic` or `iconic`? Was the name changed too?

Comment (mkArtakMSFT) [inline src/Components/Samples/BlazorServerApp/wwwroot/css/open-iconic/README.md:4]: Should we still refer to `Iconic` given that it's not there?
Highlighted below?
Or should we just change this text to not refer to `Iconic` any more for comparison.
```suggestion
### Open Iconic is the open source sibling of *[IONIC](http://ionic.io)*. It is a hyper-legible collection of 223 icons with a tiny footprint&mdash;ready to use with Bootstrap and Foundation. [View the collection](https://ionic.io/ionicons)
```

Comment (mkArtakMSFT) [reply src/Components/Samples/BlazorServerApp/wwwroot/css/open-iconic/README.md:4]: Unless these readmes were copied from the appropriate version of the framework.

**Review by blowdart — APPROVED** [review]
As a stopgap measure this works for me

**Comment by mkArtakMSFT** [comment]
@blowdart , @javiercn I wonder if we should use the suggestion from https://github.com/iconic/open-iconic/issues/60#issuecomment-1250223672 instead?

> @Mike-E-angelo Maybe you can use archive.org as a work around for now: https://web.archive.org/web/20220506153325/https://useiconic.com/icons/
> 
> But maybe the details page are not up-to-date, but better then nothing.

**Comment by blowdart** [comment]
I think that's worse, as we're relying on another 3rd party caching a 3rd party. While I generally trust archive.org page copyright owners can get things removed.

The best option would be update the whole thing, icons, readme, license. If we can't get into 7.0 rtm then we have more time and that's the way to go, this is just a tactical option right now.

**Comment by danmoseley** [comment]
Note this also exists in dotnet/samples, dotnet/maui-samples, dotnet/blazor-samples, dotnet/aspnetcore.docs. I guess those don't need updating on the GA timeline though.
https://cs.github.com/?q=org%3Adotnet+useiconic

**Comment by danroth27** [comment]
The original links are to "useiconic.com" not "useionic.com". Ionic is a product that happens to have its own set of icons, but I don't think it's related to Iconic or open iconic, which is what we use. I don't think we should be replacing links to Iconic with links to Ionic icons (ionicons).

**Review by danroth27 — APPROVED** [review]
LGTM

**Comment by mkArtakMSFT** [comment]
Issue covering remaining work: https://github.com/dotnet/aspnetcore/issues/44606

**Comment by masterparser** [comment]
I think that the website has changed to: (https://www.appstudio.dev/app/OpenIconic.html). This new site features the same number of icons, 223, and same licenses as were found at the previous site (useiconic.com/open).

**Pull request merged**